covalent_dispatcher/_cli/service.py

```diff
@@ -386,20 +388,55 @@ def status() -> None:
 
 
 @click.command()
-def purge() -> None:
+@click.option("-H", "--hard", is_flag=True, help="Perform a hard purge, deleting the DB as well")
+@click.option("-y", "--yes", is_flag=True, help="Approve without showing the warning")
+def purge(hard: bool, yes: bool) -> None:
     """
-    Shutdown server and delete the cache and config settings.
+    Shuts down the server, deletes the cache, the logs, the config settings and if desired, the database.
```

Comment:
Docstring text: "Purge Covalent from this system. This command is for developers."